src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Ticket.php

```diff
@@ -38,31 +37,26 @@ class Ticket
     /** @var string */
     private $reporter = 'sp-dashboard';
     /** @var string */
-    private $summary;
+    private $entityName;
+    /** @var string */
+    private $applicantName;
+    /** @var string */
+    private $applicantEmail;
 
-    public function __construct($summary, $description, $entityId)
+    public function __construct($entityId, $nameEn, $applicantName, $applicantEmail)
     {
-        $this->summary = $summary;
-        $this->description = $description;
         $this->entityId = $entityId;
+        $this->entityName = $nameEn;
+        $this->applicantName = $applicantName;
+        $this->applicantEmail = $applicantEmail;
     }
 
     public static function fromManageResponse($entity, Applicant $applicant)
     {
         $entityId = $entity['data']['entityid'];
```

Comment:
The name of the entity doesn't have to be translated?
I also don't like the conversion from manage arrays here, because later on it will take a lot of work if this needs to be refactored later on. For example if the names of the entities should be translated later on. I think it will be prone to bugs because you don't know what this method really will expect. I don't think we have to change it now but we have to take this into account while building further on it.

Reply:
1. No, it already is translated in Manage / SP Dashboard. I simply choose to display the English translation as English is the main language of the application.
2. Yes, I'm with you on this one. https://www.pivotaltracker.com/story/show/162378398

Reply:
:+1: